src/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackers.vue

```diff
@@ -132,26 +161,8 @@ export default {
       };
     });
 
-    const http = new APIService(
-      'https://5875ebe4-9692-48a9-90dc-b79dac993a70.mock.pstmn.io',
-      '/api/v2/Performance/EmployeeTrackers',
-    );
-    const {jsDateFormat} = useDateFormat();
-    const {locale} = useLocale();
-
-    const trackerNormalizer = data => {
-      return data.map(item => {
-        return {
-          ...item,
-          modifiedDate: formatDate(parseDate(item.modifiedDate), jsDateFormat, {
-            locale,
-          }),
-          addedDate: formatDate(parseDate(item.addedDate), jsDateFormat, {
-            locale,
-          }),
-        };
-      });
-    };
+    const api = 'api/v2/performance/employee-trackers';
```

Comment:
```js
    'api/v2/performance/employees/trackers'
```
or
```js
    'api/v2/performance/employees/{empNumber}/trackers'
```
just check these route naming. adapt if possible

Reply:
changed to ` api/v2/performance/employees/trackers`